### DISCUSSION TITLE
AAD Connect Connected Indication

### DISCUSSION THREAD

**Discussion by DarrenChap** [KelvinTegelaar/CIPP#49] [Ideas]
Hi Kelvin et al,

Originally put this in the intro section.  Doh! GitHub Gumby here....

Just wondering if displaying the users sync status with AD in the user lists and/or the Tenant selection would be handy for those of us not yet able to break away completely from on prem AD?

**Comment by KelvinTegelaar** [comment]
Good call, added to the feature requests.

**Comment by gavsto** [comment]
@DarrenChap I've added this in for you now!

**Reply by DarrenChap** [reply]
1 hour turn around wow! Thanks guys.
